ecosystem/typescript/sdk/src/providers/aptos_client.ts

```diff
@@ -308,6 +316,87 @@ export class AptosClient {
     return builder.build(payload.function, payload.type_arguments, payload.arguments);
   }
 
+  /**
+   * Generates a fee payer transaction that can be signed and submitted to chain
+   *
+   * @param sender the sender's account address
+   * @param payload the transaction payload
+   * @param fee_payer the fee payer account
+   * @param secondarySignerAccounts an optional array of the secondary signers accounts
+   * @returns a fee payer raw transaction that can be signed and submitted to chain
+   */
+  async generateFeePayerTransaction(
+    sender: MaybeHexString,
+    payload: Gen.EntryFunctionPayload,
+    feePayer: AptosAccount,
```

Comment:
One last thing, both of these should be just the address.

Then each individual user should sign with their AptosAccount